netlify.toml

```diff
@@ -7,3 +7,33 @@
   from = "/careers/*"
   to = "/careers"
   status = 200
+
+
+[[redirects]]
+    from = "/docs/api/elements"
```

Comment:
Not sure what these redirects are supposed to do, they seem to be redirecting to the same location?

Reply:
it's a mistake from a stupid bot I made 

(it's actually useful if properly configured :sweat_smile: )